dist/ba_root/mods/features/profanity.py

```diff
@@ -1,60 +1,47 @@
-# ported from ankit scripts 
-# need to update in future with easy to add custom list and more deep analysis .
-# working on other features rn, will update this later , for now lets use this
-import re
-
-PATTERN = (
-    r"fu+c+k|"
-    r"fu+c+($|)|"
-    r"fu+k+($|)|"
-    r"\w*ph+u*c+k\w*\b|"
-    r"\b\w+ch+o+d|"
-    r"randi+|"
-    r"chu+t\w*\b|"
-    r"chh+a+k+[ae]|"
-    r"hijd\w|"
-    r"lund\b|"
-    r"\bass\b|"
-    r"asshole|"
-    r"bi*tch|"
-    r"cock|"
-    r"\bga+nd\b|"
-    r"ga+ndu|"
-    r"tharki|"
-    r"tatti|"
-    r"lod\w\b|"
-    r"jha+nt|"
-    r"pu+s+y|"
-    r"pu+z+y|"
-    r"di+c+k|"
-    r"\b([mb]+c+)+\b|"
-    r"\b[mb]+[^a-zA-Z]?c+\b|"
-    r"f.u.c.k|"
-    r"b\w*s\w?d\w?k|"
-    r"m.{0,4}d.?a.{0,8}c.?h.?o.?d|"
-    r"b.+n.?c.?h.?o.?d|"
-    r"cunt|"
-    r"my+r+e|"
-    r"th+y+r|"
-    r"th+y+i+r|"
-    r"th+aa+y+o+l+i|"
-    r"th+a+y+o+l+i|"
-    r"ku+nn+a+n|"
-    r"na+y+i+n+t+e|"
-    r"pu+ll+u|"
-    r"la+(u|v)+d+\w\b|"
-    r"chu+d\w*\b|"
-    "sex+($|)|"
-    r"bo+b(s|z)|"
-    r"po+r+n|"
-    r"ni+p+le+"
-)
+# Simple yet powerful tools to censor bad words from chat messages
+# It doesn't use regex but string comparison, so hopefully it will be faster
+# With a built-in long list of profanity words (in English), there’s no need to define it yourself
+# You can add your custom words for better coverage
+# The censor can be bypassed by adding any character(s) to the words, unfortunately :(
+# But hey, at least it can attempt to censor bad words written in fancy letters :)
+# Don’t forget to install the unidecode and better-profanity modules
+# Brought to you by goblogger
+
+from better_profanity import profanity
```

Comment:
where better_profanity module is defined?

Reply:
it's a package. I guess it should be downloaded to be used.